app/code/Magento/Review/Api/Data/ReviewInterface.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Review\Api\Data;
+
+/**
+ * @api
+ */
+interface ReviewInterface
+{
+    /**#@+
+     * Constants defined for keys of  data array
+     */
+    const REVIEW_ID = 'review_id';
+    const STORE_ID = 'store_id';
+    const STORES = 'stores';
+    const TITLE = 'title';
+    const DETAIL = 'detail';
+    const NICKNAME = 'nickname';
+    const CUSTOMER_ID = 'customer_id';
+    const RATINGS = 'ratings'; // TODO: consider using extension attributes
```

Comment:
Since `Ratings` functionality is part of `Review` module there is no need to define extension attributes. In general extension attributes can help when functionality of one module needs to be extended with the functionality of the other module.